Add docs for coupon credits (#114)
spheronFdn/docs: pages/billing/credits.mdx

import { Callout } from "nextra-theme-docs";
import Navigation from "components/navigation";

# Coupon Credits

You can use coupon credits to buy bonuses.

1. Navigate to the "Coupon Credits" section under the **Billing** tab.
2. Click "Add Credit".
3. Enter the credit coupon code, complete the CAPTCHA verification and Click "Redeem Credit".

> 4. You can view the status of all your credits here, categorized as Active, Pending, and Expired.

Reviewer: mention how to use these credits as well. like add upgrade plan with credits, add ad-ons with credits and topup compute credit with credits

Reviewer: add these section and then we are good

import { Callout } from "nextra-theme-docs";
import Navigation from "components/navigation";

# Coupon Credits

Spheron credits reflect our commitment to provide users with enhanced payment flexibility and streamlined transactional processes.
One of the standout attributes of the Spheron Credits is their universal applicability.
Single credit activation code can be deployed across multiple organizations on our platform.

> You can use coupon credits to add bonuses for free.

Reviewer: not for free bro, it'll deduct from credit

1. Navigate to the "Coupon Credits" section under the **Billing** tab.
2. Click "Add Credit".
3. Enter the credit coupon code, complete the CAPTCHA verification and Click "Redeem Credit".
4. You can view the status of all your credits here, categorized as Active, Pending, and Expired.

In scenarios where a user possesses both Spheron and Referral credits, our system simplifies the experience
by representing them as one cohesive payment method. To prioritize and optimize the utilization of credits,
referral credits will be exhausted before tapping into the Spheron Credits during any transaction.

<Callout type="info">
  **NOTE:** <br /> It's pivotal to remember that while credits can be spread
  across numerous organizations, they all draw from a singular, unified credit
  pool.
</Callout>

<Callout type="info">
  **NOTE:** <br /> Each credit has an expiry. Once your credit expires, you will
  lose access to all the bonuses purchased with those credits.
</Callout>

## Upgrading Plans with Credits

1. Navigate to the "My Plan" section under the **Billing** tab.
2. Click "Change or upgrade plan" and Select your desired plan.
3. Choose **Spheron Credit** in Select Payment Method.

> 4. Read the transaction details and Click "Upgrade Plan" to initiate the payment. Upon successful deduction from credit, the plan will be upgraded to Pro.

Reviewer: ```suggestion
4. Read the payment details and Click "Upgrade Plan" to initiate the payment. Upon successful deduction from credit, the plan will be upgraded to Pro.
```

by representing them as one cohesive payment method. To prioritize and optimize the utilization of credits,
referral credits will be exhausted before tapping into the Spheron Credits during any transaction.

<Callout type="info">
  **NOTE:** <br /> It's pivotal to remember that while credits can be spread
  across numerous organizations, they all draw from a singular, unified credit
  pool.
</Callout>

<Callout type="info">
  **NOTE:** <br /> Each credit has an expiry. Once your credit expires, you will
  lose access to all the bonuses purchased with those credits.
</Callout>

## Upgrading Plans with Credits

1. Navigate to the "My Plan" section under the **Billing** tab.
2. Click "Change or upgrade plan" and Select your desired plan.
3. Choose **Spheron Credit** in Select Payment Method.
4. Read the transaction details and Click "Deposit." If you have already deposited the funds, you can move to **Step 8**.
5. To make sure your subscription is renewed every month, you need to deposit funds to the Spheron escrow wallet.
6. Add the amount you want to approve and Click "Approve." This amount should be more than or equal to the plan's Token-Value (total amount).

> 7. Add the amount you want to deposit and Click "Deposit." Make sure it is not more than the approved amount, or else the transaction will fail.

Reviewer: why is this step needed for credit payment? did you not tried it yourself??? Credit shouldn't need user to deposit any token to escrow wallet

> 8. Click "Upgrade Plan" to initiate the transaction. Upon successful transaction, the plan will be upgraded to Pro.

Reviewer: ```suggestion
8. Click "Upgrade Plan" to initiate the payment. Upon successful deduction from credit, the plan will be upgraded to Pro.
```

## Adding Add-ons with Credits

1. Navigate to the "Add-on" section under the **Billing** tab.
2. Click "Buy Add-on" and Select the type of Add-on and the Add-on Amount.
3. To add more add-on fields, Click "Add Add-on Field".
4. After adding all the add-on fields, Click "Review."
5. Choose **Spheron Credit** in Select Payment Method.

> 6. Read the transaction details, and Click "Add" to initiate the transaction.

Reviewer: ```suggestion
6. Read the add-on's payment details, and Click "Add" to initiate the payment.
```

> 7. Upon successful transaction, the Add-on will be added.

Reviewer: ```suggestion
7. Upon successful payment, the Add-on will be added.
```

## Topping up Compute Credits with Credits

1. Navigate to the "Compute Usage" section under the **Billing** tab.

> 2. Click "Top Up" under **Balance** and Select `Compute Credits` in Add-on and the Amount you wish to add.

Reviewer: ```suggestion
2. Click "Top Up" under **Balance** and Select `Compute Credits` in Add-on and the Amount you wish to add and click on Review.
```

> 3. To add more add-on fields, Click "Add Add-on Field".

Reviewer: Remove this as this is not possible. there is no other add-on fields on the compute anyways

Author: We should remove the `Add Add-on Field` button also.

> 4. After adding all the add-on fields, Click "Review."

Reviewer: remove this step

5. Choose **Spheron Credit** in Select Payment Method.

> 6. Read the transaction details, and Click "Add" to initiate the transaction.

Reviewer: ```suggestion
6. Read the add-on's payment details, and Click "Add" to initiate the payment.
```

> 7. Upon successful transaction, the Add-on will be added.

Reviewer: ```suggestion
7. Upon successful payment, the Add-on will be added.
```
